src/status_im/ui/components/list_item/views.cljs

```diff
@@ -1,82 +1,284 @@
 (ns status-im.ui.components.list-item.views
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
```

Comment:
the defview here is not necessary, but anyway when you require even macros use the alias of the namespace and don't refer directly: `views/defview`, `views/letsubs`. I know that there is many places where it isn't done but that is old code, we are trying to  be explicit everywhere now

Reply:
Understood. I will update it.

Reply:
done